models/payment/src/payment-status/transformers.ts

```diff
@@ -0,0 +1,22 @@
+import { Reference } from '@commercetools-test-data/commons';
+import { Transformer } from '@commercetools-test-data/core';
+import type { TPaymentStatus, TPaymentStatusGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TPaymentStatus, TPaymentStatus>('default', {
+    buildFields: ['state'],
+  }),
+  rest: Transformer<TPaymentStatus, TPaymentStatus>('rest', {
+    buildFields: ['state'],
```

Comment:
`state` needs to be transformed into a `Reference` like line 17, I suppose you can do the same thing since the model doesn't exist yet.